styles/widgets/generic/actionButton.generic.less

```diff
@@ -0,0 +1,66 @@
+.dx-size-default() {
```

Comment:
We do not need this block if the FAB has the same styles in default and compact size schemes.